HelpSource/Classes/NotificationCenter.schelp

```diff
@@ -4,9 +4,35 @@ related:: Classes/SimpleController, Classes/NodeWatcher
 categories:: Control
 
 description::
-One common OOP pattern is Model-View-Controller where one object (the controller) is a dependant of the model. Every time the model changes it notifies all of its dependants. In this case the model has a dictionary of dependants and iterates through those.
+One common OOP pattern is the Observer Pattern where one object (the controller or observer) is a dependant of a model or subject object ( https://en.wikipedia.org/wiki/Observer_pattern ). Every time the model changes it notifies all of its dependants. In the default implementation of this pattern in SuperCollider, the model has a dictionary of dependants and iterates through those.  A limitation of this implementation is that all dependants always receive the same message "update" when the model changes.  If a depenant is required to perform different actions depending on the type of change that happens in the model, or on the dependant itself, then this must be coded in the update method.  The NotificationCenter overcomes this limitation by making it possible to register a function to be performed for each combination of model, notification message and dependant.  There are thus 4 objects involved in each connecetion between model and dependant in the NotificationCenter:
```

Comment:
suggestion: –>

If a dependant is required to perform different actions depending on the type of change that happens in the model, or on the dependant itself, then this must be specified in the update method of the dependant.

Reply:
Hello Julian, I have integrated your suggestion in the help file of NotificationCenter and pushed the commit to my fork of supercollider git repo.  The commit is 933c52ee582e823c6dda3204cfbb1c8d24cf12b8 

I do not know how to send this commit to the supercollider/supercollider repo without opening another pull request. I will ask advice from Nathan who is maintaining the instructions for contributions on the wiki.